server/claim/claim.controller.ts

```diff
@@ -209,11 +230,11 @@ export class ClaimController {
     }
 
     @IsPublic()
-    @Get("api/claim/:id")
+    @Get("api/:namespace?/claim/:id")
```

Comment:
question: won't that create a conflict and allow the following endpoint? `:namespace?/api/:namespace?/claim/:id`. You can find that in the mapped endpoints in the startup log.